engine/verification/assigner/engine.go

```diff
@@ -63,36 +73,159 @@ func (e *Engine) Done() <-chan struct{} {
 	return e.unit.Done()
 }
 
-func (e *Engine) SubmitLocal(event interface{}) {
-	e.Submit(e.me.NodeID(), event)
+// handleExecutionReceipt receives a receipt that appears in a finalized container block. In case this verification node
+// is staked at the reference block of this execution receipt's result, chunk assignment is done on the execution result, and
+// the assigned chunks' locators are pushed to the chunks queue, which are made available for the chunk queue consumer (i.e., the
+// fetcher engine).
+func (e *Engine) handleExecutionReceipt(receipt *flow.ExecutionReceipt, containerBlockID flow.Identifier) {
+	receiptID := receipt.ID()
+	resultID := receipt.ExecutionResult.ID()
+	referenceBlockID := receipt.ExecutionResult.BlockID
+
+	log := log.With().
+		Hex("receipt_id", logging.ID(receiptID)).
+		Hex("result_id", logging.ID(resultID)).
+		Hex("reference_block_id", logging.ID(referenceBlockID)).
+		Hex("container_block_id", logging.ID(containerBlockID)).
+		Logger()
+
+	// verification node should be staked at the reference block id.
+	ok, err := e.stakedAtBlockID(referenceBlockID)
+	if err != nil {
+		log.Error().Err(err).Msg("could not verify stake of verification node for result at reference block id")
+		return
+	}
+	if !ok {
+		log.Warn().Msg("node is not staked at reference block id, receipt is discarded")
+		return
+	}
+
+	// chunk assignment
+	chunkList, err := e.chunkAssignments(context.Background(), &receipt.ExecutionResult)
+	if err != nil {
+		log.Error().Err(err).Msg("could not determine chunk assignment")
+		return
+	}
+	log.Info().
```

Comment:
We are print Info log for every receipt, I would suggest to remove this log, and convert this [Debug log](https://github.com/onflow/flow-go/pull/427/files#diff-e7a28ce3f305916ae5d3992051dab0e249b4b7f155047057774b08199c9d37e8R149) into Info log.

So that we will print Info log for processing every block.

Reply:
Refactored the log level for processing block in https://github.com/onflow/flow-go/pull/427/commits/b992381ac664511ee7052d01f224882d4bd92060. Regarding the receipt's log level, however, I would suggest keeping it at `info` level for basically two reasons (1) we are not enabling debug level logs on mainnet (2) based on my experience on debugging VNs, this particular log that shows the chunk assignment is done (with the number of assigned chunks) shown very helpful.